odl/trafos/wavelet.py

```diff
@@ -41,6 +41,60 @@
 _SUPPORTED_IMPL = ('pywt',)
 
 
+def pywt_wbasis(wbasis):
+    """Convert wbasis to `pywt.Wavelet` if it is not already one."""
+    if isinstance(wbasis, pywt.Wavelet):
+        return wbasis
+    else:
+        return pywt.Wavelet(wbasis)
+
+
+def pywt_pad_mode(pad_mode, pad_const):
+    """Convert ODL style pad mode to pywt pad mode.
+
+    Parameters
+    ----------
+    pad_mode: {'constant', 'symmetric', 'periodic', 'order0', 'order1',
+               'pywt_per'}
+       ODL style boundary condition.
+
+    Returns
+    -------
+    pywt_pad_mode: string
+        PyWavelets style boundary condition. See `pywt.MODES.modes`.
+
+        ``'constant'`` and ``pad_const==0`` : ``'zpd'``
+
+        ``'order0'`` : ``'cpd'``
+
+        ``'order1'`` : ``'sp1'``
+
+        ``'symmetric'`` : ``'sym'``
+
+        ``'periodic'`` : ``'ppd'``
+
+        ``'pywt_per'`` : ``'per'``
+    """
+    pad_mode, pad_mode_in = str(pad_mode).lower(), pad_mode
+    pad_const, pad_const_in = float(pad_const), pad_const
+
+    if pad_mode == 'constant' and pad_const == 0:
+        return 'zpd'
+    elif pad_mode == 'order0':
+        return 'cpd'
+    elif pad_mode == 'order1':
+        return 'sp1'
+    elif pad_mode == 'symmetric':
+        return 'sym'
+    elif pad_mode == 'periodic':
+        return 'ppd'
+    elif pad_mode == 'pywt_per':
+        return 'per'
+    else:
+        raise ValueError("unknown pad_mode ('{}') and pad_const ({})"
```

Comment:
no parentheses
Probably a check for `'constant'` pad mode and constant != 0 would give a better error message.